services/gcp/cloudsql/databases.yml

```
@@ -0,0 +1,31 @@
+name: Google Cloud SQL
+description: >-
+  The Cloud SQL database is the set of software and files that operates the MySQL, PostgreSQL, or SQL Server database service.
+scope: CRITICAL
+notes: >-
+  Cloud SQL is used to store and serve sensitive and application-critical data. Breach of a cloud SQL database can lead to exfiltration of highly sensitive data, or interruption of mission-critical applications.
+privileges:
+  create:
+    risks: [impact:spend, impact:consumption]
+    notes: >-
+      This privilege enables users to create databases within a Cloud SQL instance. It involves the potential excessive costs due to resource usage.
```

Grammar is a bit funky:
```suggestion
      This privilege enables users to create databases within a Cloud SQL instance. Adding databases can increase spend.
```